jscomp/core/js_fold.ml

```diff
@@ -232,7 +232,7 @@ class fold =
           let _self = _self#block _x4 in
           _self
       | Continue _x0 ->
-          let _self = _self#label _x0 in
+          let _self = option (fun _self -> _self#label) _self _x0 in
```

Comment:
I  don't know what the code does, but I've done it the same way as for other variants and it compiled.